### PR TITLE
Azure phase 2: LA

### PR DESCRIPTION
**Pull request by micheleRP**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/2214
Review deadline: Mon, July 29

## Page previews

- [What's New in Redpanda Cloud](https://deploy-preview-597--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/whats-new-cloud/)
- [Create a BYOC Cluster on Azure](https://deploy-preview-597--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/create-byoc-cluster-azure/) 
- [Tiers and Regions](https://deploy-preview-597--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/byoc-tiers/)
- [Cloud IAM Policies: Azure](https://deploy-preview-597--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/security/authorization/cloud-iam-policies-azure)
- [Cloud Overview](https://deploy-preview-597--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/cloud-overview/)
- [Azure network services](https://deploy-preview-597--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/security/cloud-security-network/)
- Azure Private Link coming in separate PR

## Checks

- [x] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

**Comment by ncole** [comment]
### [Create a BYOC Cluster on Azure](https://deploy-preview-597--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/create-byoc-cluster-azure/)
* `Ensure that you have enough quota in your subscription for DPLDSv5 and LSv3 Family vCPUs`
  * Not sure how this is going to play out, this information is no longer accurate (is now "Standard LASv3" and "Standard Dadsv5").  If it changes in the future how do we keep it up to date?

### [Create a Dedicated Cluster on Azure](https://deploy-preview-597--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/create-dedicated-cloud-cluster-aws/)
* Not completely tested, but doc is appropriate

### [Tiers and Regions](https://deploy-preview-597--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/byoc-tiers/)
* Also `ukwest` (for Azure) (also the up-to-date question)

### [Cloud IAM Policies: Azure](https://deploy-preview-597--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/security/authorization/cloud-iam-policies-azure)
* Current:
  ```
    actions = [
      "Microsoft.Resources/subscriptions/resourcegroups/read",
      "Microsoft.Storage/storageAccounts/blobServices/containers/delete",
      "Microsoft.Storage/storageAccounts/blobServices/containers/read",
      "Microsoft.Storage/storageAccounts/blobServices/containers/write",
      "Microsoft.Storage/storageAccounts/blobServices/generateUserDelegationKey/action",
      "Microsoft.Network/dnszones/read",
      "Microsoft.Network/dnszones/write",
      "Microsoft.Network/dnszones/delete",
      "Microsoft.Network/dnszones/SOA/read",
      "Microsoft.Storage/storageAccounts/read",
      "Microsoft.ContainerService/managedClusters/read",
      "Microsoft.ContainerService/managedClusters/delete",
      "Microsoft.ContainerService/managedClusters/write",
      "Microsoft.ContainerService/managedClusters/agentPools/read",
      "Microsoft.ContainerService/managedClusters/agentPools/write",
      "Microsoft.ContainerService/managedClusters/agentPools/delete",
      "Microsoft.ContainerService/managedClusters/agentPools/upgradeNodeImageVersion/action",
      "Microsoft.ContainerService/managedClusters/listClusterUserCredential/action",
      "Microsoft.Network/virtualNetworks/read",
      "Microsoft.Network/virtualNetworks/subnets/join/action",
      "Microsoft.Network/virtualNetworks/subnets/read",
      "Microsoft.Network/virtualNetworks/subnets/write",
      "Microsoft.Network/virtualNetworks/subnets/delete",
      "Microsoft.Authorization/roleAssignments/read",
      "Microsoft.Authorization/roleAssignments/write",
      "Microsoft.Authorization/roleAssignments/delete",
      "Microsoft.Authorization/roleDefinitions/write",
      "Microsoft.Authorization/roleDefinitions/read",
      "Microsoft.Authorization/roleDefinitions/delete",
      "Microsoft.ManagedIdentity/userAssignedIdentities/read",
      "Microsoft.ManagedIdentity/userAssignedIdentities/write",
      "Microsoft.ManagedIdentity/userAssignedIdentities/delete",
      "Microsoft.ManagedIdentity/userAssignedIdentities/federatedIdentityCredentials/read",
      "Microsoft.ManagedIdentity/userAssignedIdentities/federatedIdentityCredentials/write",
      "Microsoft.ManagedIdentity/userAssignedIdentities/federatedIdentityCredentials/delete",
      "Microsoft.Storage/storageAccounts/read",
      "Microsoft.Storage/storageAccounts/write",
      "Microsoft.Storage/storageAccounts/delete",
      "Microsoft.Storage/storageAccounts/blobServices/read",
      "Microsoft.Storage/storageAccounts/blobServices/write",
      "Microsoft.Network/publicIPAddresses/read",
      "Microsoft.Network/publicIPAddresses/write",
      "Microsoft.Network/publicIPAddresses/delete",
      "Microsoft.Storage/storageAccounts/queueServices/read",
      "Microsoft.Storage/storageAccounts/fileServices/read",
      "Microsoft.Storage/storageAccounts/fileServices/shares/read",
      "Microsoft.Storage/storageAccounts/listkeys/action"
    ]
    data_actions = [
      "Microsoft.Storage/storageAccounts/blobServices/containers/blobs/delete",
      "Microsoft.Storage/storageAccounts/blobServices/containers/blobs/read",
      "Microsoft.Storage/storageAccounts/blobServices/containers/blobs/write",
      "Microsoft.Storage/storageAccounts/blobServices/containers/blobs/move/action",
      "Microsoft.Storage/storageAccounts/blobServices/containers/blobs/add/action"
    ]
  ```
### [Cloud Overview](https://deploy-preview-597--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/cloud-overview/)
* No change

File: modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-azure.adoc [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+[NOTE]
+==== 
+* If you plan to create a private network in your own VPC, select the region where your VPC is located.
+* Three availability zones provide two backups in case one availability zone goes down.
+====
++ 
+Optionally, click *Advanced settings* to specify up to five key-value custom tags. After the cluster is created, the tags are applied to all Azure resources associated with this cluster. For more information, see the https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/tag-resources[Azure documentation^].
+
+. Click *Next*.
+. On the Network page, enter the connection type: either *Public* or *Private*. For BYOC clusters, *Private* is best-practice.
```
Comment (chapmanc) [inline modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-azure.adoc]: I am not sure if you want to mention it but we won't have support for peering out of the box so you might want to mention that the only supported path for the time being is private link. Just a thought

File: modules/deploy/partials/cloud/iam-policies.adoc [hunk 2]
```diff
@@ -744,3 +744,67 @@ Redpanda cluster resources:
...
 endif::[]
+
+ifdef::env-azure[]
+== Azure IAM policies
+
+IAM policies are assigned to deployed Redpanda agents for BYOC Azure
+clusters that use the following Azure services:
+
+  actions = [
+    "Microsoft.Resources/subscriptions/resourcegroups/read",
```
Comment (chapmanc) [inline modules/deploy/partials/cloud/iam-policies.adoc]: These are just slightly out of date FYI: https://github.com/redpanda-data/cloudv2/blob/main/pkg/provisioners/azure/terraform/agent/iam.tf#L16

File: modules/deploy/partials/cloud/tiers.adoc [hunk 3]
```diff
@@ -68,6 +69,18 @@ Amazon Web Services (AWS)::
 | us-west-2
 |===
 --
+Azure::
++
+--
+|=== 
+| Region 
+
+| eastus
```
Comment (chapmanc) [inline modules/deploy/partials/cloud/tiers.adoc:85]: We have a lot of zonal failures on eastus. @c4milo and @ncole should we remove eastus from this list for now?

File: modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-azure.adoc [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+** Resources:
++
+*** Microsoft.Compute
+*** Microsoft.ManagedIdentity
+*** Microsoft.Storage
+*** Microsoft.KeyVault
+*** Microsoft.Network
+*** Microsoft.Compute.EncryptionAtHost
++
+** Quota: A tier 1 Redpanda cluster requires the following:
```
Comment (t-eckert) [inline modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-azure.adoc]: Linking to [this page](https://learn.microsoft.com/en-us/azure/quotas/view-quotas) may help the reader understand quotas and how to check them.

File: modules/deploy/partials/cloud/iam-policies.adoc (hunk 2 above)
Comment (micheleRP) [reply modules/deploy/partials/cloud/iam-policies.adoc]: > These are just slightly out of date FYI: https://github.com/redpanda-data/cloudv2/blob/main/pkg/provisioners/azure/terraform/agent/iam.tf#L16

Thanks @chapmanc, I've copied over the latest!

File: modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-azure.adoc (hunk 4 above)
Comment (micheleRP) [reply modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-azure.adoc]: thanks @t-eckert, I added a link

File: modules/deploy/partials/cloud/tiers.adoc (hunk 3 above)
Comment (ncole) [reply modules/deploy/partials/cloud/tiers.adoc:85]: ^ @bpraseed

File: modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-azure.adoc (hunk 4 above)
Comment (micheleRP) [reply modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-azure.adoc]: @t-eckert or @ncole: Can you provide the quotas required for tiers 2 and 3 that we can document [here](https://deploy-preview-597--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/create-byoc-cluster-azure/#prerequisites)?

File: modules/deploy/partials/cloud/tiers.adoc (hunk 3 above)
Comment (micheleRP) [reply modules/deploy/partials/cloud/tiers.adoc:85]: @bpraseed please confirm that we support these Azure regions: eastus, norwayeast, uksouth

File: modules/deploy/pages/deployment-option/cloud/security/cloud-security-network.adoc [hunk 5]
```diff
@@ -176,3 +176,20 @@ address range of the destination VPC.
...
 Cloud network and only accept traffic from Kafka connectors that connect to
 your internal data stores to retrieve or push data.
+
+== Azure network services
+
+Redpanda Cloud supports the following Azure network services:
+
+* Time synchronization
++
+To ensure time synchronization, Redpanda Cloud in Azure uses https://learn.microsoft.com/en-us/azure/virtual-machines/windows/time-sync[Microsoft time sync^].
```
Comment (c4milo) [inline modules/deploy/pages/deployment-option/cloud/security/cloud-security-network.adoc]: maybe we should use the Linux counterpart? https://learn.microsoft.com/en-us/azure/virtual-machines/linux/time-sync

File: modules/deploy/pages/deployment-option/cloud/security/cloud-security-network.adoc [hunk 6]
```diff
@@ -176,3 +176,20 @@ address range of the destination VPC.
...
+
+* Domain Name System (DNS)
++
+Redpanda Cloud creates a new DNS zone for each cluster in the control plane
+and delegates its management exclusively to each cluster's data plane. In turn,
+the data plane creates a managed zone in Cloud DNS, managing DNS records for
```
Comment (c4milo) [inline modules/deploy/pages/deployment-option/cloud/security/cloud-security-network.adoc]: Cloud DNS -> Azure DNS

File: modules/deploy/pages/deployment-option/cloud/security/cloud-security-network.adoc [hunk 7]
```diff
@@ -176,3 +176,20 @@ address range of the destination VPC.
...
+Redpanda services, as needed. All the interactions with Cloud DNS are
+controlled by IAM policies targeted to the specific Cloud DNS resources managed
```
Comment (c4milo) [inline modules/deploy/pages/deployment-option/cloud/security/cloud-security-network.adoc]: AIM policies -> Azure RBAC policies

File: modules/deploy/partials/cloud/tiers.adoc (hunk 3 above)
Comment (c4milo) [reply modules/deploy/partials/cloud/tiers.adoc:85]: what type of failures?

File: modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-azure.adoc [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+*** Microsoft.Storage
+*** Microsoft.KeyVault
+*** Microsoft.Network
+*** Microsoft.Compute.EncryptionAtHost
++
+** Role: The Azure user must have the *Owner* role.
++
+** Quotas: Redpanda clusters requires the following quota. For more information, see the https://learn.microsoft.com/en-us/azure/quotas/view-quotas[Microsoft documentation^].
++
+*** Standard Lasv3-series vCPUs: 24
```
Comment (c4milo) [inline modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-azure.adoc:25]: This varies depending on the throughput tier, no? @bpraseed

File: modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-azure.adoc [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+*** Standard Dadsv5-series vCPUs: 8
+
+== Create a BYOC cluster
+
+To create a Redpanda cluster in your virtual private cloud (VPC), log in to https://cloud.redpanda.com[Redpanda Cloud^].
```
Comment (c4milo) [inline modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-azure.adoc]: For Azure, VPC -> Virtual Network

File: modules/deploy/pages/deployment-option/cloud/security/cloud-encryption.adoc [hunk 10]
```diff
@@ -10,9 +10,11 @@ For data on disk, Redpanda Cloud relies on the cloud provider's default volume
...
 either per disk or data chunk, depending on the cloud provider.
 
-For details about how default data at rest encryption works, see
-https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ssd-instance-store.html[AWS SSD instance store volume^]
-or https://cloud.google.com/docs/security/encryption/default-encryption[GCP default encryption at rest^].
+For details about how default data at rest encryption works, see:
+
+* https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ssd-instance-store.html[AWS SSD instance store volume^]
+* https://cloud.google.com/docs/security/encryption/default-encryption[GCP encryption at rest^]
+* https://learn.microsoft.com/en-us/azure/security/fundamentals/encryption-atrest[Azure data encryption at rest^]
```
Comment (c4milo) [inline modules/deploy/pages/deployment-option/cloud/security/cloud-encryption.adoc:17]: Azure encryption at rest? to make it consistent with GCP's, or vice versa is OK too.

File: modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-azure.adoc [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+:page-cloud: true
+
+NOTE: BYOC on Azure is currently in a limited availability (LA) release. Additional functionality is expected when it is generally available.
+
+== Prerequisites
+
+* A minimum version of `rpk` v24.1
+* A dedicated Azure subscription with the following: 
++
+** Resources:
```
Comment (c4milo) [inline modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-azure.adoc]: we should probably link the Azure documentation about how to do this: https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/resource-providers-and-types

File: modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-azure.adoc [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
++
+*** Microsoft.Compute
+*** Microsoft.ManagedIdentity
+*** Microsoft.Storage
+*** Microsoft.KeyVault
+*** Microsoft.Network
+*** Microsoft.Compute.EncryptionAtHost
++
+** Role: The Azure user must have the *Owner* role.
```
Comment (c4milo) [inline modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-azure.adoc]: The Azure user must have the *Owner* role in the subscription.

File: modules/deploy/partials/cloud/tiers.adoc (hunk 3 above)
Comment (micheleRP) [reply modules/deploy/partials/cloud/tiers.adoc:85]: added [centralus](https://deploy-preview-597--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/byoc-tiers/?tab=tabs-1-azure)

File: modules/deploy/pages/deployment-option/cloud/security/cloud-security-network.adoc (hunk 5 above)
Comment (micheleRP) [reply modules/deploy/pages/deployment-option/cloud/security/cloud-security-network.adoc]: Thanks @c4milo! I've incorporated your feedback. Please give it all a final review!

File: modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-azure.adoc [hunk 13]
```diff
@@ -0,0 +1,51 @@
...
+** Role: The Azure user must have the _Owner_ role in the subscription.
++
+** Resources: For more information, see the https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/resource-providers-and-types[Microsoft documentation].
++
+*** Microsoft.Compute
+*** Microsoft.ManagedIdentity
+*** Microsoft.Storage
+*** Microsoft.KeyVault
+*** Microsoft.Network
+*** Microsoft.Compute.EncryptionAtHost
```
Comment (RafalKorepta) [inline modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-azure.adoc:21]: I don't see `*** Microsoft.Compute.EncryptionAtHost` in the resource providers list. Should we omit this to not confuse users?

Comment (RafalKorepta) [reply modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-azure.adoc:21]: NVM, it's not registered by default.

Comment (RafalKorepta) [reply modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-azure.adoc:21]: Could we add additional link to https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/preview-features?tabs=azure-portal or mention that EncryptionAtHost is not a provider, but feature within Compute provider?

Comment (RafalKorepta) [reply modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-azure.adoc:21]: Ok, I don't know where to find `EncryptionAtHost` it's not in preview-feature page in Azure portal 🤷

Comment (RafalKorepta) [reply modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-azure.adoc:21]: If there is FAQ somewhere maybe 

When there is new subscription in azure user can see the following error
```
2024-07-30T13:41:34.944Z	ERROR	.rpk.managed-byoc	validation/validate.go:72	Checking compute... FAILED
2024-07-30T13:41:34.944Z	ERROR	.rpk.managed-byoc	validation/validate.go:74	Microsoft.Compute resource provider is not registered
2024-07-30T13:42:02.089Z	ERROR	.rpk.managed-byoc	validation/validate.go:72	Checking machine type and cpu availability... FAILED
2024-07-30T13:42:02.089Z	ERROR	.rpk.managed-byoc	validation/validate.go:74	Machine type family quota standardLASv3Family not available for Standard_L8as_v3.
2024-07-30T13:42:02.089Z	ERROR	.rpk.managed-byoc	validation/validate.go:74	Machine type family quota standardDADSv5Family not available for Standard_D2ads_v5.
2024-07-30T13:42:02.089Z	ERROR	.rpk.managed-byoc	validation/validate.go:74	Machine type family quota standardDSv3Family not available for Standard_D2s_v3.
2024-07-30T13:42:03.323Z	INFO	.rpk.managed-byoc	validation/validate.go:68	Checking policies... PASSED
2024-07-30T13:42:04.169Z	ERROR	.rpk.managed-byoc	validation/validate.go:54	An error occurred while checking network usage. GET https://management.azure.com/subscriptions/b820aa73-736b-447c-88d0-4fdf6d42187c/providers/Microsoft.Network/locations/norwayeast/usages
--------------------------------------------------------------------------------
RESPONSE 409: 409 Conflict
ERROR CODE: SubscriptionHasNoUsages
--------------------------------------------------------------------------------
{
  "error": {
    "code": "SubscriptionHasNoUsages",
    "message": "Subscription b820aa73-736b-447c-88d0-4fdf6d42187c has no usages in NRP.",
    "details": []
  }
}
--------------------------------------------------------------------------------
```

As suggested by @c4milo user need to register providers

File: modules/deploy/pages/deployment-option/cloud/whats-new-cloud.adoc [hunk 14]
```diff
@@ -7,6 +7,10 @@ This page lists new features added in Redpanda Cloud.
 
 == July 2024
 
+=== BYOC on Azure: limited availability
+
+Redpanda now supports xref:deploy:deployment-option/cloud/create-byoc-cluster-azure.adoc[BYOC clusters on Azure]. This is a limited availability release for BYOC clusters. Additional functionality is expected when this is generally available.
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/whats-new-cloud.adoc]:
```suggestion
Redpanda now supports xref:deploy:deployment-option/cloud/create-byoc-cluster-azure.adoc[BYOC clusters on Azure]. This is a limited availability release for BYOC clusters.
```

Comment (Feediver1) [reply modules/deploy/pages/deployment-option/cloud/whats-new-cloud.adoc]: The deleted text seems implicit, and we never say anything about forthcoming releases or functionality. I don't see this statement as adding value.

File: modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc [hunk 15]
```diff
@@ -40,12 +40,12 @@ You can also contact https://redpanda.com/try-redpanda?section=enterprise-trial[
...
 === Bring Your Own Cloud (BYOC)
 
-With BYOC clusters, you deploy Redpanda in your own cloud (AWS or GCP), and all data is
+With BYOC clusters, you deploy Redpanda in your own cloud (AWS, GCP, or Azure), and all data is
 contained in your own environment. This provides an additional layer of security and isolation. When you create a BYOC cluster, you select the supported xref:deploy:deployment-option/cloud/byoc-tiers.adoc[tier] that meets your compute and storage needs. Redpanda handles provisioning, operations, and maintenance. See also: <<BYOC architecture>>.
 
 ==== Sign up for BYOC
 
-To start using BYOC, contact https://redpanda.com/try-redpanda?section=enterprise-trial[Redpanda sales^] to request a private offer, and you can be billed either directly or through Google Cloud Marketplace or AWS Marketplace. 
+To start using BYOC, contact https://redpanda.com/try-redpanda?section=enterprise-trial[Redpanda sales^] to request a private offer. You are billed directly or through Google Cloud Marketplace or AWS Marketplace.
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc]:
```suggestion
To start using BYOC, contact https://redpanda.com/try-redpanda?section=enterprise-trial[Redpanda sales^] to request a private offer. You are billed directly, or through Google Cloud Marketplace or AWS Marketplace. 
```

File: modules/deploy/partials/cloud/tiers.adoc [hunk 16]
```diff
@@ -19,6 +19,7 @@ When you create a BYOC cluster, you select your usage tier. Each tier provides t
 
 [NOTE]
 ====
+* On Azure, tiers 1-3 are currently supported.
```
Comment (Feediver1) [inline modules/deploy/partials/cloud/tiers.adoc]:
```suggestion
* On Azure, tiers 1-3 are supported. 
```

Comment (Feediver1) [reply modules/deploy/partials/cloud/tiers.adoc:22]: I'm curious why you used a table in the tabs here? Only one column suggests that each tab could simply be a list, no?

File: modules/deploy/partials/cloud/tiers.adoc [hunk 17]
```diff
@@ -74,6 +75,18 @@ Amazon Web Services (AWS)::
...
+| Region 
+
+| centralus
+| eastus 
+| norwayeast
+| uksouth
+|===
+--
 ====
 endif::[]
```
Comment (Feediver1) [inline modules/deploy/partials/cloud/tiers.adoc:92]: I'm curious why you used a table in the tabs here? Only one column suggests that each tab could simply be a list, no?

**Review by Feediver1 — APPROVED** [review]
LGTM - Thanks @micheleRP

**Comment by Feediver1** [comment]
@bpraseed has advised that there are additional UI updates in the pipeline, and that he is awaiting them (should land today, 7/31) to ensure that the doc is accurate/in sync with these late UI updates. Consequently, this PR will not be included in today's release of the Cloud docs for the 7/31 Cloud release.
Once @bpraseed gets the updates and can review/approve this content, this PR will be merged. If those updates and the PM review and approval do not occur before the official GA date of August 6th, then proceed with the merge. Any additional updates would have to be captured in a separate doc issue. Thx.

File: modules/deploy/pages/deployment-option/cloud/security/authorization/cloud-iam-policies-azure.adoc
Comment (bpraseed) [inline modules/deploy/pages/deployment-option/cloud/security/authorization/cloud-iam-policies-azure.adoc:1]: Lets also highlight that the Azure permissions are scoped at resource group level. more details in this thread https://redpandadata.slack.com/archives/C0191NM4WU9/p1721922014274469?thread_ts=1721892954.039979&cid=C0191NM4WU9

**Review by bpraseed — APPROVED** [review]
LGTM. Feel free to release it and address I am permission comment later

**Pull request merged**